Mustachio/Parser.cs

```diff
@@ -16,15 +16,42 @@ public class Parser
         /// <summary>
         /// Parse the template content, producing a function that can be used to apply variables to the template. 
         /// The provided function can be reused (i.e. no state will "leak" from one application of the function to the next).
+        /// Default parsing options will be used.
+        /// </summary>
+        /// <param name="template">The content of the template to be parsed.</param>
+        /// <returns></returns>
+        public static Func<IDictionary<String, object>, String> Parse(string template)
+        {
+            return Parse(template, new ParsingOptions());
+        }
+
+        /// <summary>
+        /// This function is now obsolete. You should use the function accepting a ParsingOptions type instead.
+        /// Parse the template content, producing a function that can be used to apply variables to the template. 
+        /// The provided function can be reused (i.e. no state will "leak" from one application of the function to the next).
+        /// Other parsing options will have default values.
         /// </summary>
         /// <param name="template">The content of the template to be parsed.</param>
         /// <param name="disableContentEscaping">In some cases, content should not be escaped (such as when rendering text bodies and subjects in emails). 
         /// By default, we use content escaping, but this parameter allows it to be disabled.</param>
         /// <returns></returns>
-        public static Func<IDictionary<String, object>, String> Parse(string template, bool disableContentEscaping = false)
+        [Obsolete]
```

Comment:
@vladsandu You can include the recommended solution for using this as the description property of the `[Obsolete()]` attribute. This helps because it will show up in most IDEs as a warning, as opposed to just using the XML docblock.